src/CoreWCF.WebHttp/src/CoreWCF/UriTemplateVariableQueryValue.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Specialized;
+using System.Text;
+using CoreWCF.Runtime;
+
+namespace CoreWCF
+{
+    internal class UriTemplateVariableQueryValue : UriTemplateQueryValue
+    {
+        readonly string _varName;
+
```

Comment:
Missing accessibility modifier

Reply:
This is addressed in 181703cd67d3b8c432a4e39a9f48e927277626e4.